src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
```

Comment:
PullProofAI

The `example` property in the `ApiProperty` decorator should be a string, not an array. It should represent a single example value, not multiple values.

Recommendation:
Update the `example` property to be a single string instead of an array to align with the expected behavior of the `ApiProperty` decorator.

```diff
-     example: ['practice1', 'practice2'],
+     example: 'practice1',
```


<!-- This is an auto-generated comment by PullProofAI -->